graphql_api/types/owner/owner.graphql

```diff
@@ -37,4 +37,5 @@ type Owner {
   ): RepositoryConnection! @cost(complexity: 25, multipliers: ["first", "last"])
   username: String
   yaml: String
+  tokensRequierd: Boolean 
```

Comment:
Typo

Reply:
yep, when user is not part of the org (decorator rules)

Reply:
Fixed 👍